systemd/systemd.go

```diff
@@ -208,24 +213,27 @@ func MockJournalctl(f func(svcs []string, n int, follow bool) (io.ReadCloser, er
 type Systemd interface {
 	// DaemonReload reloads systemd's configuration.
 	DaemonReload() error
+	// DaemonReload reloads systemd's configuration if required
+	// adding flag is services are added(true) or removed(false), serviceNames list of services
```

Comment:
s/is/if

or maybe "adding: true if services are added, false otherwise".

Reply:
Fixed